client/src/featureform/cli.py

```diff
@@ -173,6 +173,9 @@ def dash():
 @click.option("--dry-run",
               is_flag=True,
               help="Checks the definitions without applying them")
+@click.option("--poll-status",
```

Comment:
This should probably be --wait or --async, --poll-status seems pretty non-standard and is focused on the implementation

Reply:
Will change to wait

Reply:
Cool, you haven't done yet so unresolving so we don't forget

Reply:
Also rn is seems like this flag doesnt do anything. Should have it set wait=True if the flag is passed